tests/test_skeptic_reviewer_queue.py

```
@@ -0,0 +1,105 @@
+)
+def test_invoke_reviewer_dispatches_vendor_contracts(
+    monkeypatch, reviewer, parent_env, expected_cmd, expects_stdin
+):
+    seen = {}
+
+    class Completed:
+        returncode = 0
+        stdout = "review output"
+        stderr = ""
+
+    def fake_run(cmd, *, input, env, **kwargs):
```

Ruff (0.16.2)

[error] Function argument `input` is shadowing a Python builtin

(A002)